From 2b5b32ade2f8ee9543c6813af42fbba0a12bb3fd Mon Sep 17 00:00:00 2001
From: Alexander Forsch <alexander.forsch@bytesatwork.ch>
Date: Fri, 15 Nov 2024 11:48:09 +0100
Subject: [PATCH] arm64: bytedevkit_imx8mp_defconfig: Fix sdma and 802.11
 firmware errors

Load sdma and 802.11 drivers as modules to avoid firmware loading errors
on boot.

Signed-off-by: Alexander Forsch <alexander.forsch@bytesatwork.ch>
---
 arch/arm64/configs/bytedevkit_imx8mp_defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/bytedevkit_imx8mp_defconfig b/arch/arm64/configs/bytedevkit_imx8mp_defconfig
index 3991c029566e..9dd82811d079 100644
--- a/arch/arm64/configs/bytedevkit_imx8mp_defconfig
+++ b/arch/arm64/configs/bytedevkit_imx8mp_defconfig
@@ -179,7 +179,7 @@ CONFIG_BT_HCIUART_BCM=y
 CONFIG_BT_HCIUART_QCA=y
 CONFIG_BT_HCIVHCI=y
 CONFIG_BT_NXPUART=m
-CONFIG_CFG80211=y
+CONFIG_CFG80211=m
 CONFIG_NL80211_TESTMODE=y
 CONFIG_CFG80211_WEXT=y
 CONFIG_MAC80211=y
@@ -862,7 +862,7 @@ CONFIG_BCM_SBA_RAID=m
 CONFIG_FSL_EDMA=y
 CONFIG_FSL_QDMA=m
 CONFIG_FSL_EDMA_V3=y
-CONFIG_IMX_SDMA=y
+CONFIG_IMX_SDMA=m
 CONFIG_MV_XOR_V2=y
 CONFIG_MXS_DMA=y
 CONFIG_MXC_PXP_V3=y
